specification.md

```diff
@@ -1490,46 +1570,34 @@ This would set the data output of the particular state to:
 
 [Switch state](#Switch-State) [conditions](#switch-state-dataconditions) require for expressions to be resolved to a boolean value (true / false). 
 
-We can now get back to our previously defined "IsAdultApplicant" expression function and reference it:
-
-```json
-{
-  "dataConditions": [ {
-    "condition": "${ fn:IsAdultApplicant }",
-    "transition": "StartApplication"
-  }]
-}
-```
-
-As previously mentioned, expressions are evaluated against certain subsets of data. For example 
-the `parameters` param of the [functionRef definition](#FunctionRef-Definition) can evaluate expressions 
-only against the data that is available to the [action](#Action-Definition) it belongs to.
-One thing to note here are the top-level [workflow definition](#Workflow-Definition-Structure) parameters. Expressions defined
-in them can only be evaluated against the initial [workflow data input](#Workflow-Data-Input).
+#### Defined Expression Functions
 
-For example let's say that we have a workflow data input of:
+In our workflow model we can define our reusable expression function:
 
 ```json
 {
-   "inputVersion" : "1.0.0"
+"functions": [
+  { 
+    "name": "IsAdultApplicant",
+    "operation": "${.applicant | .age > 18}",
+    "type": "expression"
+  }
+]
 }
 ```
 
-we can use this expression in the workflow "version" parameter:
+The function name `IsAdultApplicant` can be referenced anywhere where expressions are allowed.
+Its return type is boolean, so it makes a useful condition:
 
 ```json
 {
-   "id": "MySampleWorkflow",
```

Comment:
why are we removing this example w/o some sort of clear description . maybe im looking at diff wrong but if its removed its another ghost removal

Reply:
The version field, or any workflow header, can't be set by an expression. It's not within any state.